microsoft/botframework-solutions
# [Templates][TypeScript] Ignore completed-doc rule

## Description

- Ignore `complete-doc` TSLint rule for `Enterprise Bot Template`, `Skill Template`
- Update `Enterprise Bot` generated sample

Close #1108 

## Testing Steps

1. Open `AI\templates\Enterprise-Template\src\typescript\generator-botbuilder-enterprise`
2. Run `npm link` to symlink the package folder
3. Open a terminal wherever you want to create a bot
4. Run `yo botbuilder-enterprise`
5. Follow the prompts
6. Check that the generator finished successfully

## Checklist
\-
